docs/release_notes/0.61.0.md

```diff
@@ -0,0 +1,12 @@
+# Release 0.61.0
+## Features
+
+- Add ability to set a custom subnet when creating a nodegroup (#4025 #4044)
+
+## Bug Fixes
+
+- Set new selector session when creating nodegroup (#3993)
+
+## Acknowledgments
+Weaveworks would like to sincerely thank:
```

Comment:
shall we remove this?

Reply:
if https://github.com/weaveworks/eksctl/pull/4056 gets merged we will need to add in this person :smile: 

but otherwise yeah you are right we should remove it :smile:

Reply:
nevermind! it'll be needed when #4056 is merged!